admin_guide/diagnostics_tool.adoc

```diff
@@ -175,21 +173,18 @@ run.
 [[admin-guide-diagnostics-tool-client-environment]]
 == Running Diagnostics in a Client Environment
 
-You can access the diagnostics tool as an ordinary user, as a *cluster-admin*
-user, and can run on a host where {product-title} master or node servers are
-operating. The diagnostics attempt to use as much access as the user has
-available.
+The diagnostics tool runs using the level of permissions granted to the 
+account from which you run it. For example, as an ordinary user or a `cluster-admin` user. 
```

Comment:
"For example, as an ordinary user or a `cluster-admin` user. " is a fragment. I think you can remove it without impacting the meaning of this section. Please do that or work it into the previous sentence.